.eslintrc.js

```diff
@@ -79,6 +79,15 @@ module.exports = {
         Cypress: false,
       },
     },
+    {
+      files: ["www/**/*"],
+      rules: {
+        // We need to import React to use the Fragment shorthand (`<>`).
+        // When we use theme-ui's JSX pragma, it lists React as an unused var
+        // even though it's still needed.
+        "no-unused-vars": ["error", { varsIgnorePattern: "React" }],
+      },
+    },
```

Comment:
Could we instead of ignoring `React` use `<React.Fragment>` instead of shorthand (at least in files that use jsx pragma)? Not a blocker for this PR, just food for thought.

Reply:
For reference for other folks checking this out - this is workaround for https://github.com/emotion-js/emotion/issues/1549 / https://github.com/yannickcr/eslint-plugin-react/issues/2156

Reply:
I can make an issue for it and assign it to the community!

Reply:
When I ran this locally on `www/src` (after commenting out this override) I got 7 errors:
```
yarn eslint --ext .js,.jsx,.ts,.tsx www/src
yarn run v1.21.0
$ /mnt/d/dev/gatsby-mbp/node_modules/.bin/eslint --ext .js,.jsx,.ts,.tsx www/src

/mnt/d/dev/gatsby-mbp/www/src/components/docs-markdown-page.js
  3:8  error  'React' is defined but never used  no-unused-vars

/mnt/d/dev/gatsby-mbp/www/src/components/guidelines/nav-item.js
  3:8  error  'React' is defined but never used  no-unused-vars

/mnt/d/dev/gatsby-mbp/www/src/components/layout.js
  3:8  error  'React' is defined but never used  no-unused-vars

/mnt/d/dev/gatsby-mbp/www/src/components/showcase-details.js
  3:8  error  'React' is defined but never used  no-unused-vars

/mnt/d/dev/gatsby-mbp/www/src/pages/plugins.js
  3:8  error  'React' is defined but never used  no-unused-vars

/mnt/d/dev/gatsby-mbp/www/src/templates/template-docs-markdown.js
  3:8  error  'React' is defined but never used  no-unused-vars

/mnt/d/dev/gatsby-mbp/www/src/views/creators/index.js
  3:8  error  'React' is defined but never used  no-unused-vars

✖ 7 problems (7 errors, 0 warnings)
```

(some of them don't even use `<>` fragments)

I don't have strong opinion on ignoring this rule vs forcing to use long-hand (`<React.Fragment>`) - I do however see having to use longer form annoying (on top of devs not being aware of this problem being confused by those eslint errors) so I'd rather track `eslint-plugin-react` issue and remove the override once it's fixed upstream

Reply:
Sure, that's fine as well.